HLTrigger/JetMET/plugins/HLTScoutingPFProducer.cc

```diff
@@ -263,7 +263,7 @@ void HLTScoutingPFProducer::fillDescriptions(edm::ConfigurationDescriptions &des
   desc.add<double>("pfJetEtaCut", 3.0);
   desc.add<double>("pfCandidatePtCut", 0.6);
   desc.add<double>("pfCandidateEtaCut", 5.0);
-  desc.add<int>("mantissaPrecision", 23);
+  desc.add<int>("mantissaPrecision", 10);  //default float16, change to 23 for float32
```

Comment:
Just to note that if the comment would expressed as
```suggestion
  desc.add<int>("mantissaPrecision", 10)->setComment("default float16, change to 23 for float32");
```
it would be shown by `edmPluginHelp -p HLTScoutingPFProducer`.